lib/bolt/node/winrm.rb

```diff
@@ -428,6 +431,12 @@ def process_from_extension(path)
           'puppet.bat',
           ['apply', "\"#{path}\""]
         ]
+      else
```

Comment:
Do you want to consider adding `.py` to the list of defaults? Seems common enough?

Reply:
Sure. I considered it, hadn't made a decision so I went with the conservative option.

Reply:
I think I'll leave it out for now without more demand. It should be fairly clear what's needed to add it when it doesn't work.